### ISSUE TITLE
[FEATURE REQUEST] Have planeflight_mod recognize species names for non-fullchem simulations

### ISSUE THREAD

**Issue by yantosca** [geoschem/geos-chem#528]
In `GeosCore/planeflight_mod.F90`, there is code to translate a species name (like O3) to its species number.
```F90
       !===========================================================
       ! Species: listed as "O3", "C2H6", etc.
       ! PVAR offset: 0
       !===========================================================
       CASE DEFAULT

          ! Skip if not full-chemistry
          IF ( IS_FULLCHEM ) THEN

             ! Loop over all species
             ! match w/ species as read from disk
             DO M = 1, State_Chm%nSpecies

                ! Get info about this species from the species database
                SpcInfo => State_Chm%SpcData(M)%Info

                IF ( TRIM( SpcInfo%Name ) == TRIM( LINE ) ) THEN
                   PVAR(N) = M
                   EXIT
                ENDIF

                ! Free pointer
                SpcInfo => NULL()

             ENDDO

             ! Special flag for RO2 species
             IF ( TRIM( LINE ) == 'RO2' ) PVAR(N) = 999

             ! Special flag for AN species FP
             IF ( TRIM( LINE ) == 'AN' ) PVAR(N) = 998

             ! Special flag for NOy species FP
             IF ( TRIM( LINE ) == 'NOy' ) PVAR(N) = 997

             ! Error check
             IF ( PVAR(N) == 0 ) THEN
                IF ( Input_Opt%amIRoot ) THEN
                   WRITE( 6, '(a)' ) 'ERROR: invalid species!'
                   WRITE( 6, 110   ) TRIM( LINE )
110                FORMAT( 'Species ', a, ' not found!' )
                   WRITE( 6, '(a)' ) 'STOP in PLANEFLIGHT!'
                   CALL GEOS_CHEM_STOP
                ENDIF
             ENDIF
          ENDIF
```
However, this code is only done for species in fullchem simulations.  Recent research efforts have started to use the planeflight diagnostic for other simulations (such as tagged CO), but this name to number look up does not work.

A simple fix would be to move the species test out of the `IF (IS_FULLCHEM)` block, such as:

```F90
       !===========================================================
       ! Species: listed as "O3", "C2H6", etc.
       ! PVAR offset: 0
       !===========================================================
       CASE DEFAULT

          ! Loop over all species
          ! match w/ species as read from disk
          DO M = 1, State_Chm%nSpecies

             ! Get info about this species from the species database
             SpcInfo => State_Chm%SpcData(M)%Info

             IF ( TRIM( SpcInfo%Name ) == TRIM( LINE ) ) THEN
                PVAR(N) = M
                EXIT
             ENDIF

             ! Free pointer
             SpcInfo => NULL()

          ENDDO

          ! Skip if not full-chemistry
          IF ( IS_FULLCHEM ) THEN

             ! Special flag for RO2 species
             IF ( TRIM( LINE ) == 'RO2' ) PVAR(N) = 999

             ! Special flag for AN species FP
             IF ( TRIM( LINE ) == 'AN' ) PVAR(N) = 998

             ! Special flag for NOy species FP
             IF ( TRIM( LINE ) == 'NOy' ) PVAR(N) = 997

          ENDIF

          ! Error check
          IF ( PVAR(N) == 0 ) THEN
             IF ( Input_Opt%amIRoot ) THEN
                WRITE( 6, '(a)' ) 'ERROR: invalid species!'
                WRITE( 6, 110   ) TRIM( LINE )
110             FORMAT( 'Species ', a, ' not found!' )
                WRITE( 6, '(a)' ) 'STOP in PLANEFLIGHT!'
                CALL GEOS_CHEM_STOP
             ENDIF
       ENDIF
```

**Comment by yantosca** [comment]
This issue is now fixed in commit 7b1c6183f8294641a9fba853e10d763640a1d478.  This will go into the 13.0.0 final release.

**Comment by yantosca** [comment]
The main branch of the geoschem/GCClassic superproject repository now points to this update. It will ship with the 13.0.0-release-candidate.
